### PR TITLE
[BUMP] Operator v0.89.0/kube-rbac-proxy v0.15.0

### PR DESCRIPTION
**Pull request by Allex1**
Operator v0.89.0/kube-rbac-proxy v0.15.0 and regenerated values schema

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
@Allex1 I think something from https://github.com/open-telemetry/opentelemetry-operator/pull/2323 is causing us pain.

```
logger.go:42: 09:07:41 | targetallocator-features | 2023-11-20 09:05:10 +0000 UTC	Warning	Pod stateful-targetallocator-b45579b65-dfbds		FailedScheduling	0/1 nodes are available: 1 node(s) didn't match Pod's node affinity/selector. preemption: 0/1 nodes are available: 1 Preemption is not helpful for scheduling.	
```

@jaronoff97 @swiatekm-sumo do you know about anything in the Operator's kind cluster that configured for this test?

**Comment by TylerHelmuth** [comment]
@Allex1 you can try bumping the  helm/kind-action action to be 1.8.0

File: kind-1.24.yaml
Comment (TylerHelmuth) [inline kind-1.24.yaml:1]: Lets move this into `.github`.

File: kind-1.24.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        nodeRegistration:
+          kubeletExtraArgs:
+            node-labels: "ingress-ready=true"
+    extraPortMappings:
+      - containerPort: 80
+        hostPort: 80
+        protocol: TCP
+      - containerPort: 443
+        hostPort: 443
+        protocol: TCP
```
Comment (TylerHelmuth) [inline kind-1.24.yaml:18]: I believe these can be removed

Comment (Allex1) [reply kind-1.24.yaml]: done

File: kind-1.24.yaml
Comment (Allex1) [reply kind-1.24.yaml:1]: done

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**